src/main/scala/scorex/core/network/SyncTracker.scala

```diff
@@ -65,12 +65,20 @@ class SyncTracker(nvsRef: ActorRef,
     }
   }
 
-  def clearStatus(remote: InetSocketAddress): Unit =
+
+  //todo: combine both?
```

Comment:
both what?

Reply:
Both lines starting with "statuses.find" and "lastSyncSentTime.find" (logic is the same: look for a peer with the given socket address and remove it from a map if found